packages/maskbook/src/plugins/Trader/constants/shashimiswap.ts

```diff
@@ -0,0 +1,27 @@
+import { ChainId, ERC20TokenDetailed } from '../../../web3/types'
+import { AMPL, SUSHI, DAI, YAM, WBTC, MSKA, MSKB, MSKC, USDC, USDT, WETH, WETH_ONLY, RUNE } from './trader'
+
+/**
+ * Some tokens can only be swapped via certain pairs,
+ * so we override the list of bases that are considered for these tokens.
+ */
+export const SASHIMISWAP_CUSTOM_BASES: {
```

Comment:
Please take confirmation to ensure those settings synced with the original front-end app of ShashimiSwap.